registry/application/services/service_publisher_service.py

```diff
@@ -329,7 +329,8 @@ def daemon_config(self, environment):
                 "service_id": service.service_id,
                 "metering_end_point": f"https://{network_name}-marketplace.singularitynet.io",
                 "authentication_addresses": [member.address for member in organization_members],
-                "blockchain_enabled": True
+                "blockchain_enabled": True,
+                "passthrough_enabled": True
```

Comment:
looks good .. do we have any test case to see the JSON o/p ?

Reply:
Added test case for daemon config for both MAIN and TEST network.